control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1804,6 +1808,25 @@ func (r *reconciler) ensureIngressControllersRemoved(ctx context.Context, hcp *h
 		log.Info("There are no ingresscontrollers, nothing to do")
 		return true, nil
 	}
+
+	// If the DeletionTimestamp reaches the gracePeriod, the controller
+	// should ignore the object and continue with the deletion
+	counter := 0
+	for i := range ingressControllers.Items {
+		ic := &ingressControllers.Items[i]
+		if ic.DeletionTimestamp != nil {
+			deletionTime := ic.ObjectMeta.DeletionTimestamp.Add(DeletionSecondsTimeout)
+			if ic.ObjectMeta.DeletionTimestamp.After(deletionTime) {
+				// ToDo (jparrill): Add a new metric to reflect the ignored cloudDanglingObjects
```

Comment:
Did you create a Jira ticket for this? If not, could you then put the ticket # in the TODO for future reference?

Reply:
I don't think is necessary, I need to delete that comment because we throw away that path in a call. Instead, we decided to use a new `.status.conditions` that only appears in case of dangling objects. I will delete the comment. Thanks